### PR TITLE
New documentation

### LINKED ISSUE #25: Font Generator
Thanks to the good work of @himedlooff, it seems that we now have a working solution to our font generator issue. Here are the new findings:
- Use FontSquirrel's [Webfont Generator](http://www.fontsquirrel.com/tools/webfont-generator) to generate the webfont version. 
- Since our glyphs are in the "Private Use Area" of the unicode, FontSquirrel's generator will subset the font and cut out the private use glyphs unless you tell it **not** to. 
- Go into the expert options and select "No subsetting".
- Generate webfont!

If you spot any bugs with this webfont generator, please post your findings.

### LINKED ISSUE #42: Document how to run the font through the web font generator
@KimberlyMunoz  found the configuration file for [FontSquirrel's webfont generator](https://github.com/cfpb/cf-icons/blob/42d7af06f24665268bbbcb878b7cbccbb3b36513/src/font-source/generator_config.txt) after checking out the repo. Let's add some wording in the README that describes what it is and how it's used, so that others know how to generate the web font files.

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
I wrote some documentation on how to run new fonts through Font Squirrel in CONTRIBUTING.MD. 

I also removed an extra link to the components section, we already link to the cf-icons and components earlier in the readme, but the extra link and headline points to the wrong documentation for people who want to get started on cf-icons.
## Review
- @Scotchester 
- @anselmbradford 
- @ascott1 

Fixes #42. Fixes #25.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,35 +1,60 @@
...
+### Uploading new fonts
+
+If you are contributing a new icon, please generate the associated webfonts
+with your new font file.
+
+We use Font Squirrel to generate webfonts.
+
+1. Add your font source file (.ttf) to `src/font-source` folder
+
+2. Visit [FontSquirrel WebFont Generator](http://www.fontsquirrel.com/tools/webfont-generator) and upload your .ttf file and our `generator_config.txt` file located in the font-source folder.
```
Comment (jimmynotjim) [inline CONTRIBUTING.md]: Should there be semantic breaks here and on L:56?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,35 +1,60 @@
...
 <https://cfpb.github.io/capital-framework/contributing/>.
+
+### Uploading new fonts
+
+If you are contributing a new icon, please generate the associated webfonts
+with your new font file.
+
+We use Font Squirrel to generate webfonts.
+
+1. Add your font source file (.ttf) to `src/font-source` folder
```
Comment (anselmbradford) [inline CONTRIBUTING.md]: Minor... you might add a little more context here:

> From the cf-icons project directory, add your font source file (.ttf) to the `src/font-source` folder.

File: CONTRIBUTING.md (hunk 1 above)
Comment (anselmbradford) [inline CONTRIBUTING.md]: Minor... You might want to inline-code style the folder name, as you do in the previous step:

> `font-source`

Alternatively, you might say:

> … and upload your .ttf file and our `generator_config.txt` file from the cf-icons project `src/font-source` folder.

Comment (KimberlyMunoz) [inline CONTRIBUTING.md]: I always forget about the semantic line breaks when I'm doing an ordered list. Added!

File: CONTRIBUTING.md (hunk 2 above)
Comment (KimberlyMunoz) [inline CONTRIBUTING.md]: :+1:

**Comment by KimberlyMunoz** [comment]
Gone ahead and made the recommended changes. Thanks!

**Comment by jimmynotjim** [comment]
LGTM. 

Not sure how everyone feels about squashing review edits down before merging, but it's something we've been doing on the FJ team to keep the log a bit cleaner.

**Comment by Scotchester** [comment]
We should probably start supporting WOFF2.

**Comment by ascott1** [comment]
This looks good to me too! Sorry it slipped through the cracks.

@Scotchester do you want to file an issue about WOFF2?

**Pull request merged**